config/environments/production.rb

```diff
@@ -131,7 +131,7 @@
   config.assets.precompile += javascript.map{|js| js + ".js"} + stylesheets + %w(background-size.htc)
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = !!SETTINGS[:require_ssl]
```

Comment:
Yeah, that'd work.  At the moment, require_ssl is used in all environments so we should probably be consistent.  I can't think of a reason to not have it in development, since it's configurable and defaults to off.